fcrepo-audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -106,6 +107,7 @@ public void process(final Exchange exchange) throws Exception {
     private static final Property RDF_TYPE = createProperty(RDF + "type");
 
     private static final String EMPTY_STRING = "";
+    private static final String RESOURCE_TYPES = "org.fcrepo.jms.resourceType";
```

Comment:
Is there a reason the `RESOURCE_TYPES` is not being defined with the other headers?
https://github.com/fcrepo4-exts/fcrepo-camel/blob/master/src/main/java/org/fcrepo/camel/JmsHeaders.java#L21